lib/structures/Invite.js

```diff
@@ -48,6 +49,20 @@ class Invite extends Base {
         this._createdAt = data.created_at !== undefined ? data.created_at : null;
         this.presenceCount = data.approximate_presence_count !== undefined ? data.approximate_presence_count : null;
         this.memberCount = data.approximate_member_count !== undefined ? data.approximate_member_count : null;
+        if(data.stage_instance !== undefined) {
+            data.stage_instance.members = data.stage_instance.members.map((m) => {
+                m.id = m.user.id;
+                return m;
+            });
+            this.stageInstance = {
+                members: this.guild.members.update(data.stage_instance.members, this.guild),
```

Comment:
This will throw [an error](https://github.com/abalabahaha/eris/blob/dev/lib/util/Collection.js#L31) since you're trying to send an array
```suggestion
                members: data.stage_instance.members,
```

Reply:
Wrong suggestion, see just pushed commit

Reply:
Well there's a [reason](https://github.com/abalabahaha/eris/pull/1216#discussion_r680408886) I didn't do it the same way as you in your pushed commit lol. Are you not able to repro that error referenced in that comment?